packages/utils/src/lib/filter.ts

```diff
@@ -0,0 +1,24 @@
+export function filterItemRefsBy<T extends { refs: object[] }>(
+  items: T[],
+  refFilterFn?: (item: T['refs'][number]) => boolean,
```

Comment:
Why is the filtering function optional? In what use case would I want to call a `filterX` function and not filter anything in it?